conda_forge_feedstock_ops/container_utils.py

```diff
@@ -107,13 +110,51 @@ def _get_proxy_mode_container_args():
     ]
 
 
+@dataclass
+class Mount:
+    host_path: Path
+    """
+    The path on the host to mount.
+    """
+    container_path: PurePosixPath
+    """
+    The path in the container to mount to.
+    """
+    read_only: bool = True
+    """
+    Whether to mount the path as read-only.
+    """
+
+    @classmethod
+    def to_cf_feedstock_ops_dir(cls, host_path: Path, read_only: bool = True):
+        """
+        Creates a new mount to the default CF_FEEDSTOCK_OPS_DIR.
+        """
+        return cls(
+            host_path=host_path,
+            container_path=CF_FEEDSTOCK_OPS_DIR,
+            read_only=read_only,
+        )
+
+    def to_mount_args(self) -> list[str]:
+        """
+        Get the corresponding mount arguments for the Docker CLI.
+        """
+        args = [
+            "--mount",
+            f"type=bind,source={self.host_path},destination={self.container_path}",
+        ]
+        if self.read_only:
+            args[-1] += ",readonly"
+        return args
+
+
 def run_container_operation(
     args: Iterable[str],
     json_loads: Callable = json.loads,
     tmpfs_size_mb: int = DEFAULT_CONTAINER_TMPFS_SIZE_MB,
-    input: Optional[str] = None,
-    mount_dir: Optional[str] = None,
-    mount_readonly: bool = True,
+    stdin_input: Optional[str] = None,
```

Comment:
Just following best practices here and not shadowing built-in `input` here.
(we probably won't change the signature again too soon)